include/RevCoProc.h

```diff
@@ -31,29 +31,35 @@
 #include "RevFeature.h"
 #include "RevMem.h"
 #include "RevInstTable.h"
+#include "RevProcPasskey.h"
+#include "RevProc.h"
 
 namespace SST::RevCPU{
-
+class RevProc;
 // ----------------------------------------
 // RevCoProc
 // ----------------------------------------
-struct RevCoProc : SST::SubComponent {
-
-  SST_ELI_REGISTER_SUBCOMPONENT_API(SST::RevCPU::RevCoProc);
+class RevCoProc : public SST::SubComponent {
+public:
+  SST_ELI_REGISTER_SUBCOMPONENT_API(SST::RevCPU::RevCoProc, RevProc*);
```

Comment:
Would using the fully qualified namespace `SST::RevCPU::RevProc*` allow you to remove the forward decl of `class RevProc` from above?